examples/complete/main.tf

```diff
@@ -30,36 +18,11 @@ module "s3_bucket" {
   s3_replication_enabled       = local.replication_enabled
   s3_replica_bucket_arn        = join("", module.s3_bucket_replication_target.*.bucket_arn)
   s3_replication_rules         = local.s3_replication_rules
+  privileged_principal_arns = var.privileged_principal_enabled ? {
```

Comment:
`terraform fmt`

Reply:
`terraform fmt` didn't catch it. Fixed.

Reply:
Actually, `terraform fmt` keeps on reverting to this. I moved the declaration of `privileged_principal_arns` to a local block in `privileged-principals.tf`.